BEIMA.Backend.Test/DeviceFunctions/UpdateDeviceTest.cs

```diff
@@ -97,8 +97,10 @@ public async Task ExistingDevice_UpdatesDevice_ReturnsOKResponse()
             Assert.IsNotNull(response);
             Assert.That(response, Is.TypeOf(typeof(OkObjectResult)));
             Assert.That(((OkObjectResult)response).StatusCode, Is.EqualTo((int)HttpStatusCode.OK));
-            var deviceBson = (Dictionary<string, object>)((OkObjectResult)response).Value;
-            Assert.That(deviceBson["serialNum"], Is.EqualTo(device.SerialNum));
```

Comment:
Do we want to get rid of this assert?

Reply:
The asserts added below check essentially the same thing.

Reply:
Okay, I didn't see serial number checked and wanted to make sure we aren't losing an assert we needed